File: 2.0/python/webhooks/createWebhookNonProfitAgency.py
from callfire.client import CallfireClient

client = CallfireClient('api-login', 'api-password')
response = client.webhooks.createWebhook(
    body={
        'name': 'Call finished webhook',
        'resource': 'OutboundCall',
        'events': ['Finished'],
        'callback': 'http://callback-service.com/webhook'
    }
).result()

# see sample JSON response for this API
# on 'curl' samples tab
print(response)

File: 2.0/php/webhooks/createWebhookNonProfitAgency.php
<?php

class ApiClientSample {

    public static function main() {
        $client =\CallFire\Api\DocumentedClient::createClient ("login", "password");
        $request = $client -> createWebhook();
        $body = '{
                    "name":"Call finished webhook",
                    "resource":"OutboundCall",
                    "events":["Finished"],
                    "callback":"http://callback-service.com/webhook"
                 }';
        $request->getOperationConfig()->setBodyParameter($body);
        $result = $client->request($request);
        $json = json_decode($result->getBody());
    }
}

ApiClientSample::main();
